src/components/Field.jsx

```diff
@@ -196,46 +188,21 @@ class FolderPickerFieldComponent extends Component {
 export const FolderPickerField = translate()(FolderPickerFieldComponent)
 
 export const CheckboxField = translate()((props) => {
-  const { value, onChange, onInput, required, label, dirty, touched, errors } = props
-  let input
-
-  if (value) {
-    input = (
-      <input
-        type='checkbox'
-        className={styles['coz-field-input-checkbox']}
-        value={value}
-        checked='checked'
-        onChange={onChange}
-        onInput={onInput}
-      />
-      )
-  } else {
-    input = (
-      <input
-        type='checkbox'
-        className={styles['coz-field-input-checkbox']}
-        value={value}
-        onChange={onChange}
-        onInput={onInput}
-      />
-      )
-  }
-
-  const conditionals = {
-    'coz-field--required': required === true,
-    'coz-field--error': errors.length !== 0,
-    'coz-field--dirty': dirty === true || touched === true
-  }
+  const { value, onChange, onInput, label, errors } = props
+  let input = <input
+    type='checkbox'
+    className={styles['coz-field-input-checkbox']}
+    value={value}
+    checked={ value }
```

Comment:
Thanks for simplifying this conditional block, but shouldn't `checked={!!value}` should be more accurate ? Also, we should try to keep consistent spacing after and before brackets.